src/Benchmarks/Benchmarks.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <PackageReference Include="Dapper" Version="$(DapperVersion)" />
+    <PackageReference Include="Jil" Version="2.15.4" />
```

Comment:
Move version number to `build/dependencies.props` and reference with a variable.